typings/sbvr-types.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module '@resin/sbvr-types' {
+	import * as Promise from 'bluebird';
```

Comment:
NIT: call this Bluebird

Reply:
We use `Promise` everywhere else so I'd rather keep the consistency, we don't use native promises because we still support node versions where native promises are slower

Reply:
I wasn't suggesting to switch to native Promises, just calling these things Bluebirds.
Let's keep it as `Promise` if it's like that everywhere.